cmd/osbuild-worker/main.go

```diff
@@ -516,6 +516,10 @@ var run = func() {
 		worker.JobTypeAWSEC2Share: &AWSEC2ShareJobImpl{
 			AWSCreds: awsCredentials,
 		},
+		worker.JobTypeImageBuilderManifest: &ImageBuilderManifestJobImpl{
+			// TODO: needs the same things as the depsolve and container
+			// resolver jobs
+		},
```

Comment:
You could add the job to the list above instead, together with the existing depsolve and search jobs, so that it can run in parallel with jobs that actually build images.

Reply:
ah of course!  ty!